### PR TITLE
Add description for the "Managing Price" topic

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds description for the "Managing Price" topic

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/catalog/pricing.html

### CODE REVIEW

File: src/catalog/pricing.md [hunk 1]
```diff
@@ -1,25 +1,27 @@
 ---
 title: Managing Price
 sections:
- - title: Advanced Pricing
-   content: The Advanced Pricing settings define the conditions required for special pricing that is available for a specific customer group or shared catalog.
-   url: /catalog/pricing-advanced.html
+  - title: Advanced Pricing
```
Comment (erikmarr) [inline src/catalog/pricing.md:4]:
```suggestion
  - title: Advanced Pricing
```

**Review by erikmarr — APPROVED** [review]

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@erikmarr the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1320`.

**Pull request merged**